src/app/shared/components/addscenario/addscenario.component.html

```diff
@@ -52,11 +52,10 @@ <h4 class="modal-title pull-left">New Scenario </h4>
                         <div class="input-invalid-warning" *ngIf="!newScenForm.get('regressionRegions.regressions.equation').valid && newScenForm.get('regressionRegions.regressions.equation').dirty">Expression
                             is required</div>
                     </div>
-
                     <!-- Equivalent Years -->
                     <div class="form-group">
                         <label class="req" for="equivalentYears">Equivalent Years (optional):</label>
-                        <input class="form-control" (keypress)="_keyPress($event)" type="number" formControlName="equivalentYears" >
+                        <input class="form-control" (keypress)="_keyPress($event)" type="number" [value]="0" formControlName="equivalentYears" >
```

Comment:
I wonder if this edit is causing the issues I mentioned in a different spot where the existing equivalentYears aren't showing up when cloning a scenario. maybe we want to check if the value exists and set it to 0 if not.